Documentation/botr/readytorun-format.md

```diff
@@ -225,22 +236,67 @@ additional data determined by the flags.
 | ``READYTORUN_FIELD_SIG_MemberRefToken``      |  0x10 | If set, the token is memberref token. If not set, the token is fielddef token. |
 | ``READYTORUN_FIELD_SIG_OwnerType``           |  0x40 | Field type. Typespec appended as additional data. |
 
-### READYTORUN_ IMPORT_SECTIONS::AuxiliaryData
+### READYTORUN_IMPORT_SECTIONS::AuxiliaryData
 
 For slots resolved lazily via ``READYTORUN_HELPER_DelayLoad_MethodCall`` helper, auxiliary data are 
-compressed argument maps that allow precise GC stack scanning while the helper is running.
+compressed argument maps that allow precise GC stack scanning while the helper is running. The CoreCLR runtime class [``GCRefMapDecoder``](https://github.com/dotnet/coreclr/blob/6b9a3d3a87825b1a34bd8f114c9b181ce75b3b2e/src/inc/gcrefmap.h#L157) is used to parse this information. This data would not be required for runtimes that allow conservative stack scanning.
+
+The auxiliary data table is supposed to contain the exact same number of GC ref map records as there are method entries in the import section. To accelerate GC ref map lookup, the auxiliary data section starts with a lookup table holding the offset of every 1024-th method in the runtime function table within the linearized GC ref map.
+
+| Offset in auxiliary data | Size | Content |
+|-------------------------:|-----:|:--------|
+| 0 | 4 | Offset to GC ref map info for method #0 relative to this byte i.e. 4 * (MethodCount / 1024 + 1) |
```

Comment:
The markdown should be pleasant to read as raw text. Could you please add spaces to these tables as necessary?

Reply:
Does this also mean we should use line breaks within paragraphs to "manually word-wrap"? I looked at various CoreCLR documents and they usually didn't; after all we probably cannot do that within tables.

Reply:
For now (as of 2nd commit) I have manually aligned the text tables except for the variable-length last column.

Reply:
>  except for the variable-length last column.

It does not look very pleasant. I would align the last column too. E.g. Would you format the code comment this way?

Reply:
Based on your suggestion I have aligned all tables in 5th commit.